tests/src/Kernel/fixtures/markup_rendering_patterns/banner.yml

```
@@ -2,18 +2,18 @@ default_page_banner_centered_with_link_full_width:
   render:
     '#type': pattern
     '#id': banner
-    '#variant': 'default'
+    '#variant': default
     '#fields':
       settings:
         'hero': false
         'shade': false
         'full_width': true
         'centered': true
-      'title': "EU Budget for the future"
-      'description': "Innovation, economy, environment and geopolitics"
+      'title': 'EU Budget for the future'
```

```suggestion
      title: 'EU Budget for the future'
```
we should use quotes for keys only when it would be invalid yml if not, like '#something' or 'div.something'
same for the others below